packages/sdks-tests/src/specs/index.ts

```diff
@@ -74,110 +74,119 @@ function isBrowser(): boolean {
+  '/custom-breakpoints-reset': { content: customBreakpointsReset },
+  '/text-block': { content: textBlock },
+  '/text-eval': { content: textEval },
+  '/state-binding': { content: stateBinding },
+  '/nested-symbols': { content: nestedSymbols },
+  '/personalization-container': { content: personalizatContainer, target: 'gen1' },
+  '/editing-styles': { content: EDITING_STYLES, isVisualEditingTest: true },
+  '/video': { content: video },
+  '/repeat-items-bindings': { content: REPEAT_ITEMS_BINDINGS },
+  '/input-default-value': { content: INPUT_DEFAULT_VALUE },
+  '/duplicate-attributes': { content: DUPLICATE_ATTRIBUTES },
+  '/js-content-is-browser': { content: JS_CONTENT_IS_BROWSER },
+  '/slot': { content: SLOT },
+  '/slot-with-symbol': { content: SLOT_WITH_SYMBOL },
+  '/slot-without-symbol': { content: SLOT_WITHOUT_SYMBOL },
+  '/no-trusted-hosts': { content: HOMEPAGE, isVisualEditingTest: true },
+  '/editing-styles-no-trusted-hosts': { content: EDITING_STYLES, isVisualEditingTest: true },
+  '/animations': { content: ANIMATIONS },
+  '/data-preview': { content: DATA_PREVIEW },
+  '/form': { content: FORM },
+  '/default-styles': { content: DEFAULT_STYLES },
+  '/css-properties': { content: CSS_PROPERTIES },
+  '/hover-animation': { content: HOVER_ANIMATION },
+  '/tabs': { content: TABS },
+  '/custom-components': { content: CUSTOM_COMPONENTS },
+  '/basic-styles': { content: BASIC_STYLES },
+  '/accordion': { content: ACCORDION },
+  '/accordion-one-at-a-time': { content: ACCORDION_ONE_AT_A_TIME },
+  '/accordion-grid': { content: ACCORDION_GRID },
+  '/symbol-tracking': { content: SYMBOL_TRACKING },
+  '/columns-with-different-widths': { content: COLUMNS_WITH_DIFFERENT_WIDTHS },
+  '/custom-components-models-show': { content: CUSTOM_COMPONENTS_MODELS_RESTRICTION },
+  '/custom-components-models-not-show': { content: CUSTOM_COMPONENTS_MODELS_RESTRICTION },
+  '/editing-box-columns-inner-layout': { content: EDITING_BOX_TO_COLUMN_INNER_LAYOUT },
+  '/with-fetch-options': { content: HOMEPAGE },
+  '/symbol-with-repeat-input-binding': { content: SYMBOL_WITH_REPEAT_INPUT_BINDING },
+  '/children-slot-placement': { content: CUSTOM_COMPONENT_CHILDREN_SLOT_PLACEMENT },
+  '/dynamic-loading': { content: DYNAMIC_LOADING_CUSTOM_COMPONENTS },
+  '/eager-dynamic-loading': { content: EAGER_DYNAMIC_LOADING_CUSTOM_COMPONENTS },
+  '/ssr-binding': { content: SSR_BINDING_CONTENT },
+  '/blocks-class-name': { content: BLOCKS_CLASS_NAME },
+  '/duplicated-content-using-nested-symbols': { content: DUPLICATED_CONTENT_USING_NESTED_SYMBOLS },
+  '/custom-components-nowrap': { content: CUSTOM_COMPONENTS_NOWRAP },
+  '/override-base-url': { content: HTTP_REQUESTS },
 } as const;
 
 export type Path = keyof typeof PAGES;
 
-const GEN1_ONLY_PATHNAMES: Path[] = ['/api-version-v1', '/personalization-container'];
-const GEN2_ONLY_PATHNAMES: Path[] = [];
-
 export const getAllPathnames = (target: 'gen1' | 'gen2'): string[] => {
```

Comment:
target can be of type `Page['target']` to include `'all'` what do you think?

Reply:
`getAllPathnames` is meant to be called within a specific e2e test app, to retrieve the pathnames that it needs to pre-render. so it doesn't make sense to allows `'all'` as an option

Reply:
ohh right, makes sense